Apps/Src/FaultState.c

```diff
@@ -79,8 +85,8 @@ void EnterFaultState(void) {
          * Temp error - Motor will throttle itself, and there's nothing we can do additional to cool it down
          */
     }
-    else if(FaultBitmap & FAULT_READBPS){ //This has been put in with future development in mind, it is not currently tripped by anything.
-        nonrecoverableFaultHandler();
+    else if(FaultBitmap & FAULT_READBPS){ // Missed BPS Charge enable message
+        killContactorHandler();
```

Comment:
You probably want to return here, otherwise you get stuck in the infinite loop below.